### DISCUSSION TITLE
MemoryError

### DISCUSSION THREAD

**Discussion by lcast5576** [pycaret/pycaret#1297] [Q&A]
**Describe the bug**

setup() is keep getting memoryerror with data shape (276759, 893)

**To Reproduce**

```python
<Paste your code here>
```

**Expected behavior**

**Additional context**

Computer memory is 16gb 
pycaret '2.2.0'
Can't share data set. 
**Versions**
<details>

</details>

**Comment by Yard1** [comment]
Please try updating PyCaret to the latest release (2.3.1). If you are using feature interactions or polynomial, trigonometric features, turn them off.

**Reply by Muhtasham** [reply]
Hello same problem here, 

I have 35gb of ram, yet still with default setting with no extra parameter getting memory issue. Is it due to size of dataset around the 100mb?

**Reply by BradKML** [reply]
I have the same issue now, when there are like 1000+ columns after modification, since there were like a few dozen categorical column that got blown up, even when the maximum amount of categories (or minimum percentage) per column is set it still causes RAM to go out-of-memory and crash silently when starting with feature selection... Maybe PyCaret defaults to subtractive approach rather than additive approach? Sorry @Yard1
